specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-04-01/examples/Registry/CodeContainer/get.json

```diff
@@ -0,0 +1,33 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "testrg123",
+    "registryName": "testregistry",
+    "api-version": "2023-04-01",
+    "codeName": "testContainer"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/testrg123/providers/Microsoft.MachineLearningServices/registries/testworkspace/codes/testContainer",
+        "name": "testContainer",
+        "type": "Microsoft.MachineLearningServices/registries/codes",
+        "properties": {
+          "description": "string",
+          "tags": {
```

Comment:
>tags [](http://example.com/codeflow?start=11&length=4)

why is tags inside of properties instead of at top level?

Reply:
ok to answer this question and the question above, we have a legacy exception to use tags at the property level. But it's only for our "assets"-code/model/etc and job types

Reply:
ARm will not be able to use these tags I dont think policy etc will work with this scheme

Reply:
Please address the latest response.